python/invoke_release/tasks.py

```diff
@@ -1045,6 +1049,67 @@ def _post_rollback(current_version, rollback_to_version):
         plugin.post_rollback(ROOT_DIRECTORY, current_version, rollback_to_version)
 
 
+def _get_version_to_bump(changelog_message):
+    major_commit_present = None
+    minor_commit_present = None
+    patch_commit_present = None
+
+    for line in changelog_message:
+        if line.startswith('- ' + MAJOR_TAG):
+            major_commit_present = True
+        elif line.startswith('- ' + MINOR_TAG):
+            minor_commit_present = True
+        elif line.startswith('- ' + PATCH_TAG):
+            patch_commit_present = True
+        else:
+            # If a line in the changelog message doesn't start with a version
+            # tag, suggest nothing.
```

Comment:
Looks like you wrapped at 80 characters. Wrap at 120, please.

Reply:
Removed in favor of you suggestion of immediatly returning "major" when a major tag is present.